src/blog_vi/templates/card.html

```diff
@@ -1,6 +1,16 @@
 <!-- Base template of the article card on the blog page. -->
+<style>
+    .legacy {
+        display: none;
+    }
+</style>
 
+{% if article.is_legacy == True %}
+<div class="legacy">
+{% endif %}
+{% if article.is_legacy == False %}
 <div class="w-full md:w-1/3 p-6 flex flex-col flex-shrink">
```

Comment:
Will it work if you add the following thing, instead of overriding the while div line?

```suggestion
<div class="w-full md:w-1/3
                     p-6 
                     flex 
                     flex-col 
                     flex-shrink"
                     {% if article.is_legacy %}legacy{% endif %}>
```